transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -315,7 +315,7 @@ static jstring netty_epoll_native_kernelVersion(JNIEnv* env, jclass clazz) {
 }
 
 static jboolean netty_epoll_native_isSupportingSendmmsg(JNIEnv* env, jclass clazz) {
-    if (sendmmsg) {
+    if (&sendmmsg) {
```

Comment:
Hmm, you are right about the warning.  When I change `if(sendmmsg)` to `if(sendmmsg != null)` I get the same warning as you do.

I agree the warning is overzealous.  Talking to other C++ people around me, the point of the warning is to protect against accidentally doing:

```
if (sendmssg()) { ... }
```

Which is apparently a common mistake.  

I have upodated the code to use !=, PTAL.